diff --git a/node_modules/@alicloud/credentials/dist/util/helper.js b/node_modules/@alicloud/credentials/dist/util/helper.js
index 0217f3b..ac64e4a 100644
--- a/node_modules/@alicloud/credentials/dist/util/helper.js
+++ b/node_modules/@alicloud/credentials/dist/util/helper.js
@@ -24,10 +24,7 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.DEFAULT_CLIENT = exports.DEFAULT_UA = void 0;
 const os = __importStar(require("os"));
-const kitx_1 = __importDefault(require("kitx"));
-const path_1 = __importDefault(require("path"));
-const pkg = kitx_1.default.loadJSONSync(path_1.default.join(__dirname, '../../package.json'));
-exports.DEFAULT_UA = `AlibabaCloud (${os.platform()}; ${os.arch()}) ` +
-    `Node.js/${process.version} Core/${pkg.version}`;
-exports.DEFAULT_CLIENT = `Node.js(${process.version}), ${pkg.name}: ${pkg.version}`;
+const package_json_1 = __importDefault(require("../../package.json"));
+exports.DEFAULT_UA = `AlibabaCloud (${os.platform()}; ${os.arch()}) ` + `Node.js/${process.version} Core/${package_json_1.default.version}`;
+exports.DEFAULT_CLIENT = `Node.js(${process.version}), ${package_json_1.default.name}: ${package_json_1.default.version}`;
 //# sourceMappingURL=helper.js.map
\ No newline at end of file
